run.sh

```diff
@@ -6,15 +6,14 @@ source ./train_environ.sh
 # add them to your PYTHONPATH:
 #export PYTHONPATH=${PYTHONPATH}:path/to/your/dependency
 
-# Comment and uncomment as required:
-# Tensorflow
-#export PYTHONPATH=${PYTHONPATH}:./tensorflow
-#python ./tensorflow/train_tensorflow.py
+# Comment and uncomment as required...
+# Tensorflow:
+python train_tensorflow.py
```

Comment:
Havent looked closely, but didnt understand why we need to do this ? 
I would choose to treat all the three approaches as first class citizens 😇

Reply:
For now, because of the paths (`model.gin` and `preprocessing.gin`), but that'll be fixed (with another `ROOT` environment variable).